src/gui/main.cpp

```diff
@@ -176,7 +176,7 @@ int main(int argc, char **argv)
 
             if (QSystemTrayIcon::isSystemTrayAvailable()) {
                 app.tryTrayAgain();
-            } else if (!app.backgroundMode()) {
+            } else if (!app.backgroundMode() && !AccountManager::instance()->accounts().isEmpty()) {
```

Comment:
Hm good point. If you start the client and no account is configured, the account wizard will show up. If you then close the wizard, there will be no way to get the wizard back with this change. On the other hand, it will just look wrong if we always show the main dialogue when opening the desktop client, because then the account wizard AND the main dialog will be shown. I think I need to find a way to open the account wizard if no account is configured in that case.

Reply:
I checked that again, and the man dialog still opens if there is no account configured, because of some logic that comes before this part. However I have now made sure that if no accounts are configured the account wizard will show up and not the main dialog.